src/neural_de.jl

```diff
@@ -183,33 +182,21 @@ function (n::NeuralDSDE{M})(x,p=n.p) where {M<:FastChain}
     solve(prob,n.args...;sensealg=TrackerAdjoint(),n.kwargs...)
 end
 
-function Lux.initialparameters(rng::AbstractRNG, n::NeuralDSDE)
-    p1 = Lux.initialparameters(rng, n.model1)
-    p2 = Lux.initialparameters(rng, n.model2)
-    return Lux.ComponentArray((p1 = p1, p2 = p2))
-end
-
-function Lux.initialstates(rng::AbstractRNG, n::NeuralDSDE)
-    st1 = Lux.initialstates(rng, n.model1)
-    st2 = Lux.initialstates(rng, n.model2)
-    return (state1 = st1, state2 = st2)
-end
-
 function (n::NeuralDSDE{M})(x,p,st) where {M<:Lux.AbstractExplicitLayer}
-    st1 = st.state1
-    st2 = st.state2
+    st1 = st.model1
+    st2 = st.model2
     function dudt_(u,p,t;st=st1)
```

Comment:
Why st=st1? This will make st a local variable and the one in the actual function scope will not be updated.